docs/02_cleos/03_command-reference/net/connect.md

```diff
@@ -1,17 +1,56 @@
+## Command
+```sh
+cleos net connect [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to connect to.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Start a new connection to a peer
+Start a new connection to a specified peer. This command allows an operator's node to connect to another peer without restarting the node.
```

Comment:
passive?

This command allows an operator's node to connect to another peer without restarting the node.

to

Use this command to

Reply:
i.e. Use this command to connect to a peer without restarting the node.

Reply:
Point taken, but I want to highlight the fact that this command is particularly useful for node operators. "Use this command" for a generic user won't stress this fact. Therefore, to achieve this and make it active voice I changed to: "A node operator can use this command..."